Content.Server/Tabletop/TabletopSystem.cs

```diff
@@ -31,10 +35,17 @@ public override void Initialize()
             SubscribeNetworkEvent<TabletopMoveEvent>(OnTabletopMove);
             SubscribeNetworkEvent<TabletopDraggingPlayerChangedEvent>(OnDraggingPlayerChanged);
             SubscribeNetworkEvent<TabletopStopPlayingEvent>(OnStopPlaying);
+            SubscribeLocalEvent<TabletopGameComponent, ActivateInWorldEvent>(OnTabletopActivate);
             SubscribeLocalEvent<TabletopGameComponent, ComponentShutdown>(OnGameShutdown);
             SubscribeLocalEvent<TabletopDraggableComponent, ComponentGetState>(GetCompState);
         }
 
+        private void OnTabletopActivate(EntityUid uid, TabletopGameComponent component, ActivateInWorldEvent args)
+        {
+            if(_actionBlockerSystem.CanInteract(args.User))
+                OpenTable(args.User, args.Target);
+        }
```

Comment:
Good change, should we just remove the verb now? Or is it better to keep it?

Reply:
I think it's fine to leave ti

Reply:
I think it's fine to leave it